### ISSUE TITLE
Playback at 432 Hz Frequency (0.981818x speed)

### ISSUE THREAD

**Issue by CubedApple** [KRTirtho/spotube#1939]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

I feel "pain" listening to music in the standard 440 Hz frequency :)

### Suggested solution

Hi,

I am a regular user of Spotube on Linux (and Windows).

Can you please add an option for a default playback speed of 0.981818x ? 

This way, I can hear my audio at 432 Hz instead of the standard 440 Hz. I would love to listen to music in the 432 Hz frequency.

Regards,
T.

### Useful resources

I believe this is the correct maths according to this VLC adjustment: https://roelsworld.eu/432-tuning/real-time-auto-pitching-video-player/

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by jojommeke** [comment]
im intrested in what difference listening on 432 hz makes

**Comment by Im-Jaydan** [comment]
> im intrested in what difference listening on 432 hz makes
Not sure if related. but, 432 hz is the earth frequency (also said to have healing effects). The Rockefellers took a part in standardizing music frequency, moving from 432hz > 440hz.

**Comment by CubedApple** [comment]
> > im intrested in what difference listening on 432 hz makes
> > Not sure if related. but, 432 hz is the earth frequency (also said to have healing effects). The Rockefellers took a part in standardizing music frequency, moving from 432hz > 440hz.

Yes, I agree with this. Thank you for sharing. 432 Hz is the frequency of all things in the universe so it only makes sense that our music is too. I am not against 440 Hz, in fact I have recorded music in 440 Hz. However, the option to listen in 432 Hz would be awesome for all. And yes, they made a move from 432 Hz to 440 Hz to keep us "out of tune" with the universal frequency.
